lib/std/start.zig

```diff
@@ -526,9 +526,11 @@ fn posixCallMainAndExit(argc_argv_ptr: [*]usize) callconv(.c) noreturn {
             @call(.always_inline, std.os.linux.pie.relocate, .{phdrs});
         }
 
-        // This must be done after PIE relocations have been applied or we may crash
-        // while trying to access the global variable (happens on MIPS at least).
-        std.os.linux.elf_aux_maybe = auxv;
+        if (!std.os.linux.extern_getauxval) {
```

Comment:
I now realize that this check would essentially have to be "if std.os.linux.getauxval was analyzed" or is *potentially* going to be called somewhere. That makes this a lot harder.

Reply:
This should probably be done through an std option.